app/views/home/feed.xml.builder

```diff
@@ -0,0 +1,19 @@
+atom_feed do |feed|
+  feed.title 'Awesome Foundation – Winning Projects'
```

Comment:
We try not to include any actual text in the templates, etc. This text should go in the `config/locales/*` files.

I'd probably just also stick with "Awesome Foundation - Projects" - I don't think we tend to refer to "winners" anywhere else on the site.